update-ngxblocker

```diff
@@ -326,7 +326,7 @@ get_options() {
 }
 
 main() {
-	local REPO=https://raw.githubusercontent.com/markwbrown/nginx-ultimate-bad-bot-blocker/master
+	local REPO=https://raw.githubusercontent.com/mitchellkrogza/nginx-ultimate-bad-bot-blocker/master
```

Comment:
Please `rebase` both commits in this `PR` into a single commit

you can see a [rebasing example on my site](https://it-offshore.co.uk/code/81-alpine-linux-pull-requests-on-github)